simulation/utils/generate_map.py

```diff
@@ -55,82 +57,101 @@ def get_seed(self):
         """Get the seed of the map"""
         return self.seed
 
-    def X(self):
+    def get_X_list(self):
         """Get list of x coordinates"""
-        xs = [point.x for point in self.points_before_interpolation] if not len(self.map_points) else [point[0] for point in self.map_points]
+        if not len(self.map_points):
```

Comment:
```suggestion
        if self.map_points.size == 0:
```
i w inicjalizacji self.map_points = np.array([]) mozna od razu

Reply:
oki,  użyję :3